devdocs/orientation-guide.md

```diff
@@ -0,0 +1,118 @@
+
+4. Screenshots (if any)
+
+
+> Note: The task descriptions are brief by intention. We want you to try to figure out how to do those things by yourself.
+
+## Orientation task list
+
+_**Phase A**: Know the project_
+
+1.  Understand what TEAMMATES is about.<>
+    * [product intro page](https://teammatesv4.appspot.com) shown to potential users
+
+    * [project vision, challenges, and principles](http://teammatesv4.appspot.com/dev/devman.html%23project)
+    * [feature overview](https://teammatesv4.appspot.com/features.html) (users’ point of view)
+
+2.  Decide in which role(s) you want to contribute. i.e., tester, copy-editor, etc. as specified [earlier in this document](#id.49lgk6iu1w59).
+3.  If you are aiming for tech writer or tester roles, get an instructor account using the link in the TEAMMATES home page. Remember to mention the purpose of your request under any other comments.
+
+    If you are sure you want to become a contributor, you can skip this step. In Phase B, you get to set up TEAMMATES on your own machine and use that instance to try the functionality.
+
+    If you plan to contribute code, continue with the tasks below.
+
+
+
+_**Phase B**: Set up locally_
+
+1.  Set up TEAMMATES development environment on your computer.  
+    Important:  
+    * Follow instructions to the letter. _Install the specified versions of the tool stack, not the latest versions._  
+    * If you encounter any problems, refer to the [Developer Troubleshooting Guide](https://www.google.com/url?qhttps://docs.google.com/document/d/1_p7WOGryOStPfTGA_ZifE1kVlskb1zfd3HZwc4lE4QQ/pub?embedded%3Dtrue&sa=D&ust=1458427634398000&usg=AFQjCNGVBlVtbfAZEnGnVwGudarEORsfRw).
+2.  Get dev green. It is OK to proceed to the next phase if you have fewer than 5 failing test cases.
+
+
+
+
+_**Phase C**: Deploy_
+
+1.  Deploy your own copy of TEAMMATES to your staging server.
+
+_**Phase D**: Tinker with the code_
+
+1.  Fork our repo at GitHub
+2.  Create a branch named ‘tinker’
+3.  In that branch, do a small change to the code that makes a visible change to TEAMMATES UI. Preferably, your change should improve the UI in some way, or at least not make it worse.
+4.  Deploy the changed version to your staging server.
+5.  Push your changes to your fork.
+6.  Create a new issue in our issue tracker with the following info  
+    Issue title: ‘New contributor self intro [your name]’    
+    a) the link to your app  
+    b) link to your fork  
+    c) a screenshot of the change you did (as seen on the UI)  
+    d) your real name and a short self-intro
+
+
+_**Phase E**: Start contributing_
+After you complete the above tasks, you are ready to become a contributor. Steps for fixing an issue is explained in the [process](https://github.com/TEAMMATES/repo/blob/master/devdocs/process.md) document. New contributors are advised to start doing an issue labeled d.FirstTimers (but do not do more than one of those), move to other issues labeled as d.Contributors  
```

Comment:
One more thing, add a line break between "Phase E: ..." and "After you complete ...".